florestad/src/main.rs

```diff
@@ -19,14 +19,18 @@
 
 mod cli;
 
+use std::sync::Arc;
 use std::time::Duration;
 
 use clap::Parser;
 use cli::Cli;
 use florestad::Config;
 use florestad::Florestad;
+use futures::executor::block_on;
+use tokio::sync::RwLock;
+use tokio::time::sleep;
 
-#[async_std::main]
+#[tokio::main]
```

Comment:
Remove this `#[tokio::main]` and make the main sync, or we get this panic:

```
thread 'main' panicked at /home/erik/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.37.0/src/runtime/scheduler/multi_thread/mod.rs:86:9:
Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.
```